shared/src/validation-schemas/podcast/create-podcast.validation-schema.ts

```diff
@@ -0,0 +1,28 @@
+import * as Joi from 'joi';
```

Comment:
rename file to ``shared/src/validation-schemas/podcast/podcast.validation-schema.ts